README.md

```diff
@@ -0,0 +1,16 @@
+# Satellite Ansible Roles
+
+Ansible Roles to setup / configure Red Hat Satellite. The roles will be created using [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) or FAM.
```

Comment:
`setup / configure` to me tends to mean install and maintain. I would prefer something more akin to the FAM language "Ansible roles for interacting with the Satellite API, combining modules into workflows" or some such.